linux/keyman-config/createkeymandirs.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/sh
-
-sudo mkdir -p /usr/local/share/keyman
-sudo mkdir -p /usr/local/share/fonts/keyman
-sudo mkdir -p /usr/local/share/doc/keyman
-sudo chmod 00777 /usr/local/share/keyman
-sudo chmod 00777 /usr/local/share/fonts/keyman
-sudo chmod 00777 /usr/local/share/doc/keyman
```

Comment:
Now that this is cleaned up, we can also clean
https://github.com/keymanapp/keyman/blob/6ced6475851e8a416a73dde599a97f2a66702e3f/docs/linux/keyman-config.md?plain=1#L38-L39